setup.py

```diff
@@ -60,132 +108,78 @@ def download_driver(zip_name: str) -> None:
 
 
 class PlaywrightBDistWheelCommand(BDistWheelCommand):
-    user_options = BDistWheelCommand.user_options + [
```

Comment:
Env var seems simpler, since option passing via `build` is not fully working anyways: https://github.com/pypa/setuptools/issues/2491